### ISSUE TITLE
How to pass a LAYOUT?

### ISSUE THREAD

**Issue by ThierryO** [paleolimbot/qgisprocess#75]
I'm having troubles to export an atlas to pdf. I'm passing the project and layout as in the example below.

```
qgis_run_algorithm(
  algorithm = "native:atlaslayouttopdf", LAYOUT = "my_layout",
  PROJECT_PATH = here(target_folder, "my_project.qgz")
)
```

Running the code returns an error. I've checked the path to the project and the name of the layout.

```
../../src/core/qgsmessagelog.cpp:29 : (logMessage) [4ms] 2021-12-29T16:19:03 Processing[2] Cannot find layout with name "my_layout"
```

**Comment by florisvdh** [comment]
Should hopefully be solved by PR #79. @ThierryO can you confirm?

**Comment by florisvdh** [comment]
Considered solved by PR #79.

**Comment by florisvdh** [comment]
In hindsight, this was probably a duplicate of https://github.com/paleolimbot/qgisprocess/issues/68.
